www/templates/elevate/index.html

```diff
@@ -161,14 +161,21 @@ <h6 class="pt-5 pb-2 mb-2 text-uppercase border-bottom">{% translate 'FAQ' %}</h
                     </div>
                     <div class="answer">
                         {% blocktranslate trimmed %}
-                            Currently ELevate provides Leapp data for migration from CentOS 7 to the following OS:
+                            Currently ELevate provides Leapp data for migration from the following OS:
+                            <ul>
+                                <li>CentOS 7</li>
+                                <li>EuroLinux 7</li>
+                                <li>Red Hat Enterprise Linux 7</li>
+                            </ul>
+                            to the following OS:
                             <ul>
                                 <li>AlmaLinux OS 8</li>
                                 <li>CentOS Stream 8</li>
                                 <li>EuroLinux 8</li>
                                 <li>Oracle Linux 8</li>
                                 <li>Rocky Linux 8</li>
                             </ul>
+                        You can automate the process with the <a href="https://github.com/EuroLinux/autoelevate/">AutoELevate project</a>. Additionally, it covers certain edge cases.
```

Comment:
Why can't it be merged into regular ELevate, especially edge cases?

Reply:
[The script](https://github.com/EuroLinux/autoelevate/blob/master/autoelevate.sh) uses the `check_supported_config` function to determine:

- that en EFI system can ELevate only to AlmaLinux - with other systems it won't boot anymore. I don't know if this issue has been resolved, therefore it's locked for safety.
- that it attempts to run on a supported Enterprise Linux 7
- that it doesn't run on a system with FIPS mode or Secure Boot

[Regular ELevate](https://wiki.almalinux.org/elevate/ELevate-quickstart-guide.html) only covers the basic steps required to turn CentOS 7 into 8.

I see no obstacles in adding the prerequisites to the site but still with a human error it's possible to brick a system with not enough caution. The script has them all covered and won't proceed once it's undesired.

Reply:
I don't like the idea of sending of the user to another repo and another script. I don't think it would benefit the community as a whole.

Reply:
In this case I can remove that line and all references to the script both here and in the Wiki and instead provide manual steps.

Reply:
ok.